CONTRIBUTING.md

```diff
@@ -104,11 +101,16 @@ black quantus/INSERT_YOUR_FILE_NAME.py
 ```bash
 flake8 quantus/INSERT_YOUR_FILE_NAME.py
 ```
-- Create `pytests` for new functionality (if needed) and add under `tests/` folder
-- If the `pytests` include a new category of `@pytest.mark` then add that category with description to `pytest.ini`
-- Make sure all unit tests are passed and the coverage level is maintained (we aim at ~100% code coverage for Quantus):
+- Create tests for new functionality and add under `tests/` folder.
+- If tests include a new category of `@pytest.mark` then add that category with description to `pytest.ini`
```

Comment:
E.g., you can run `tox run -e py38 -- -s -v`, and you will see the following output:
```
============================================================================== test session starts ===============================================================================
platform darwin -- Python 3.8.16, pytest-7.3.2, pluggy-1.0.0 -- /Users/artemsereda/Documents/IdeaProjects/Quantus/.tox/py38/bin/python
cachedir: .tox/py38/.pytest_cache
rootdir: /Users/artemsereda/Documents/IdeaProjects/Quantus
configfile: pytest.ini
plugins: mock-3.10.0, cov-4.1.0, xdist-3.3.1, lazy-fixture-0.6.3
collected 669 items 
```
I did not change any testing configuration. 
Is there a reason why you though it could be broken?